services/tables/src/main/java/com/linkedin/openhouse/tables/api/validator/impl/OpenHouseTablesApiValidator.java

```diff
@@ -308,6 +309,42 @@ public void validateCreateLock(
     }
   }
 
+  @Override
+  public void validateSearchSoftDeletedTables(String databaseId, Pageable pageable) {
+    if (databaseId == null) {
```

Comment:
There could be some common code between this PR and Levi's pagination PR. But @jiang95-dev 's PR is not yet merged.